pkg/autoscaler/autoscaler.go

```diff
@@ -146,22 +153,23 @@ type perPodAggregation struct {
 	accumulatedConcurrency float64
 	probeCount             int32
 	window                 time.Duration
-	lameduckTime           *time.Time
+	latestStatTime         *time.Time
```

Comment:
why is this a pointer?

Reply:
Use `nil` to determine whether a time has already been set.